draft-ietf-tsvwg-multipath-dccp.mkd

```diff
@@ -748,14 +751,13 @@ Smooth RTT (=3)
   by the Age parameter.
 {: vspace='0'}
 
-Age (=4)
-: The Age parameter is a 4-byte value which is set to the age or  
-  timestamp when the Ack for the datagram was received in case of  
-  RTT type = 0 and may contain the periods for computing of derived  
-  RTT values depending on other RTT types, i.e., the Minimum (=1)  
-  and Maximum (=2) as well as the averaged smoothed RTT value (=3).  
+Age
+: The Age parameter is a 4-byte value which is set to the age and  
```

Comment:
It was already said above that the Age information is a 32-bit integer so the 4-byte here is now redundant.

Reply:
Please look into https://github.com/markusa/ietf-multipath-dccp/pull/52/commits/ece3530163cf2b46fd0f8c09264a17a03b21782d